mindsdb/integrations/handlers/anthropic_handler/anthropic_handler.py

```diff
@@ -0,0 +1,112 @@
+import os
+from typing import Optional, Dict
+
+from anthropic import Anthropic, HUMAN_PROMPT, AI_PROMPT
+import pandas as pd
+
+from mindsdb.utilities.config import Config
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+from mindsdb.utilities.log import get_log
+
+
+logger = get_log("integrations.anthropic_handler")
+
+class AnthropicHandler(BaseMLEngine):
+    """
+    Integration with the Anthropic LLM Python Library
+    """
+    name = 'anthropic'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.default_chat_model = 'claude-2'
+        self.supported_chat_model = ['claude-1', 'claude-2']
```

Comment:
Nitpick, but I recommend renaming to `supported_chat_models`, plural.

Reply:
Thanks for the review, I have made the necessary changes